src/screens/FeedScreen/index.js

```diff
@@ -17,7 +18,7 @@ import {Context} from '../../context';
 import {DISCOVERY_TAB_TOPICS} from '../../utils/constants';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
 import {normalizeFontSizeByWidth} from '../../utils/fonts';
-import {setFeedByIndex, setTimer} from '../../context/actions/feeds';
+import {setFeedByIndex} from '../../context/actions/feeds';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `setTimer` import has been removed. Ensure that this function is not used elsewhere in the file or that it's been properly replaced if it was previously used.
```diff
- import {setFeedByIndex, setTimer} from '../../context/actions/feeds';
+ import {setFeedByIndex} from '../../context/actions/feeds';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->